CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.1]
+### Changed
+- Bump @metamask/utils from ^3.5.0 to ^5.0.2 ([#201](https://github.com/MetaMask/eth-json-rpc-middleware/pull/201))
+- Bump @metamask/eth-sig-util from ^5.0.3 to ^5.1.0 ([#205](https://github.com/MetaMask/eth-json-rpc-middleware/pull/205))
+- Bump eth-block-tracker from ^7.0.0 to ^7.0.1 ([#204](https://github.com/MetaMask/eth-json-rpc-middleware/pull/204))
+- Bump @metamask/eth-sig-util from ^5.0.2 to ^5.0.3 ([#203](https://github.com/MetaMask/eth-json-rpc-middleware/pull/203))
```

Comment:
From the POV of someone using this package, eth-sig-util is only getting bumped once. We can consolidate these two entries:

```suggestion
- Bump @metamask/eth-sig-util from ^5.0.2 to ^5.1.0 ([#203](https://github.com/MetaMask/eth-json-rpc-middleware/pull/203), [#205](https://github.com/MetaMask/eth-json-rpc-middleware/pull/205))
- Bump eth-block-tracker from ^7.0.0 to ^7.0.1 ([#204](https://github.com/MetaMask/eth-json-rpc-middleware/pull/204))
```

Reply:
Actually scratch that, it was bumped zero times. Those were lockfile-only changes.

Reply:
```suggestion
- Bump eth-block-tracker from ^7.0.0 to ^7.0.1 ([#204](https://github.com/MetaMask/eth-json-rpc-middleware/pull/204))
```